server/Dockerfile

```
@@ -0,0 +1,16 @@
+FROM node:10.15.3-alpine AS base
+RUN apk --no-cache update \
+  && apk --no-cache upgrade
+
+FROM base AS buildServer
+WORKDIR /app
+COPY . .
+RUN \
+  chown -v node /app/dist/index.html \
+  && chown -v node /app/dist
+
+FROM buildServer AS production
+USER node
+ENV nexus_token="invalid"
+ENTRYPOINT [ "/bin/sh" ]
+CMD [ "start.sh" ]
```

See my comment on the start.sh file. So we can't use it as CMD here : we don't have nvm (and don't care about it), we don't want to source missing env file. 

```suggestion
CMD [ "sh", "-c", "npm run build-config && npm run server" ]
```